zbus/src/connection/mod.rs

```diff
@@ -1534,6 +1534,81 @@ mod p2p_tests {
         )
     }
 
+    #[cfg(any(
+        all(feature = "vsock", not(feature = "tokio")),
+        feature = "tokio-vsock"
+    ))]
+    #[test]
+    #[timeout(15000)]
+    fn vsock_connect() {
+        crate::utils::block_on(test_vsock_connect()).unwrap();
+    }
+
+    #[cfg(all(feature = "vsock", not(feature = "tokio")))]
```

Comment:
most of the code for both `vsock` and `vsock-tokio` seems identical. Best to have one function.